fp.js

```diff
@@ -6,8 +6,8 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(function(person){
-  peoplesAges.push(person.age)
+people.map ((person) => {
```

Comment:
👍 the use of `.map` is perfect, but we need to capture the returned array into the `peoplesAges` array or we'll lose the mapped data:
```js
peoplesAges = people.map ((person) => {
```